src/holofuel/pages/Home/Home.module.css

```diff
@@ -43,51 +45,82 @@
 }
 
 .balance {
-  background-color: lightgray;
-  padding: 20px;
-  border-bottom: 1px solid black;
-  border-radius: 4px;
+  background-color: var(--color-caribbean-green);
+  padding: 46px 20px;
+  border-top-left-radius: 4px;
+  border-top-right-radius: 4px;
+}
+
+.balance-label {
+  color: white;
+  text-align: center;
 }
 
-.balance-header {
+.balance-row {
   display: flex;
   align-items: center;
 }
 
-.balance-label {
-  margin-left: auto;
-  font-size: 12px;
+.balance-amount {
+  text-align: center;
```

Comment:
The amount isn't actually centered.  Let's look into this...

Reply:
good catch, fixed.